config/src/config/consensus_config.rs

```diff
@@ -27,7 +27,7 @@ impl Default for ConsensusConfig {
         ConsensusConfig {
             contiguous_rounds: 2,
             max_block_size: 1000,
-            max_pruned_blocks_in_mem: 10000,
+            max_pruned_blocks_in_mem: 100,
```

Comment:
hard to relate the comment in the PR to the change. 10K to 100 seems a big change and it's not clear what that would affect. I am a bit confused here